support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx

```diff
@@ -199,7 +198,9 @@ function WeeklyCheckoutForm(props: PropTypes) {
 		props.setBillingCountry(props.deliveryCountry);
 	};
 
-	const inThreeTier = showThreeTierCheckout(props.participations);
+	const urlParams = new URLSearchParams(window.location.search);
```

Comment:
Ah I see, no abTest definition so need to rely upon URL Param. Probably should of been like this during the test, oh well, thanks!